packages/terra-menu/src/MenuItem.jsx

```diff
@@ -292,12 +327,22 @@ class MenuItem extends React.Component {
       }
 
       content = (
-        <Arrange
-          fitStart={fitStartIcon}
-          fill={textContainer}
-          fitEnd={hasChevron ? <ChevronIcon className={cx('chevron')} /> : null}
-          align="center"
-        />
+        <>
+          <Arrange
+            fitStart={fitStartIcon}
+            fill={textContainer}
+            fitEnd={hasChevron ? <ChevronIcon className={cx('chevron')} /> : null}
+            align="center"
+          />
+          {screenReaderResponse}
+        </>
+      );
+    } else {
```

Comment:
is the else added for handling sub-menu ..??

Reply:
This is for handling non-toggleable menus. 
https://engineering.cerner.com/terra-framework/tests/cerner-terra-framework-docs/menu/menu/default-menu 
The items in this example do not use arrange component as it is non-toggleable.